app/inbound/total/get-total-by-calculation-reference.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../../data')
+
+const getTotalByCalculationReference = async (calculationReference, transaction) => {
```

Comment:
CalculationReference is not in the Total table, getTotalByCalculationId should be the precise function name. Something like 
const getTotalByCalculationId = async (calculationId, transaction) => {

....
where: {
calculationId
}

Reply:
calculationReference and this function is prior to calculationId so this function uses the alias we set in data. At this stage the service only understands calculationReference

Reply:
renamed Reference to Id as suggested